portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -142,9 +149,8 @@ private Comment entityToComment(Entity entity){
     String title = (String) entity.getProperty("title");
     String body = (String) entity.getProperty("body");
     String email = (String) entity.getProperty("email");
-    long timestamp = (long) entity.getProperty("timestamp");
-    long id = entity.getKey().getId();
-    Comment comment = new Comment(title, body, email, timestamp, id);
+    double score = (double) entity.getProperty("score");
```

Comment:
existing comments might not have a score which might cause some problems. I would suggest either clearing out the existing table or writing some logic to handle comments without scores. (or, migrate the existing table to a new table and backfill scores on existing comments. This would be a one-off job and would absolutely be the way to do it if you had important data in the database but you could likely get away with just deleting all the data and starting over again).

Reply:
I deleted all the old comments already, so it should not be a problem anymore, right?